kyuubi-server/src/main/scala/org/apache/kyuubi/engine/spark/SparkProcessBuilder.scala

```diff
@@ -44,6 +45,10 @@ class SparkProcessBuilder(
 
   import SparkProcessBuilder._
 
+  val yarnClient = getYarnClient
+
+  def getYarnClient: YarnClient = YarnClient.createYarnClient
```

Comment:
The name of `getYarnClient` method seems to be a little confused. It looks like the `getter` method of `yarnClient` variable.

Reply:
@wForget, what about `createYarnClient`? @942011334, sorry to suggest a bit confusing method name.

Reply:
> what about createYarnClient?

LGTM